core/src/main/java/io/neow3j/protocol/core/methods/response/BufferStackItem.java

```diff
@@ -0,0 +1,85 @@
+package io.neow3j.protocol.core.methods.response;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import io.neow3j.contract.ScriptHash;
+import io.neow3j.model.types.StackItemType;
+import io.neow3j.utils.BigIntegers;
+import java.math.BigInteger;
+import java.util.Arrays;
+import java.util.Objects;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BufferStackItem extends StackItem {
+
+    @JsonProperty("value")
+    private byte[] value;
+
+    public BufferStackItem() {
+        super(StackItemType.BUFFER);
+    }
+
+    public BufferStackItem(byte[] value) {
+        super(StackItemType.BUFFER);
+        this.value = value;
+    }
+
+    /**
+     * Decodes the stack item's base64-encoded value and returns it as a byte array.
+     *
+     * @return the value of this stack item.
+     */
+    public byte[] getValue() {
+        return this.value;
+    }
+
+//    /**
+//     * <p>Gets this byte array's value as an address.</p>
+//     * <br>
+//     * <p>Expects the byte array to be a script hash in little-endian order.</p>
+//     *
+//     * @return the address represented by this byte array.
+//     */
+//    public String getAsAddress() {
+//        return new ScriptHash(getValue()).toAddress();
+//    }
```

Comment:
@gsmachado Here I am unsure, whether this should be implemented in the buffer stack item. Could you give me some input here?

Reply:
@mialbu as discussed, leave the `getAsAddress()` method. 👍